modules/oauth-api/src/main/java/org/mule/runtime/oauth/api/builder/OAuthDancerBuilder.java

```diff
@@ -60,6 +61,17 @@
    */
   OAuthDancerBuilder<D> tokenUrl(String tokenUrl, TlsContextFactory tlsContextFactory);
 
+  /**
+   * Mule, after receiving the authentication code from the OAuth server (through the redirectUrl) will call this url to get the
+   * access token.
+   *
+   * @param tokenUrl The OAuth authentication server url to get access to the token.
+   * @param tlsContextFactory References a TLS config that will be used to do HTTP request during the OAuth dance.
+   * @param proxyConfig References the proxy configuration which through the HTTP request will go during the request.
+   * @return this builder
+   */
+  OAuthDancerBuilder<D> tokenUrl(String tokenUrl, TlsContextFactory tlsContextFactory, ProxyConfig proxyConfig);
```

Comment:
add a default implementation to not break the api (call the other `tokenUrl` method without the proxy)